### PR TITLE
Feature/sr 7 implement navbar

### PR DESCRIPTION
**Pull request by GalacticOctopus**
Also added ability to check if user is authenticated

### CODE REVIEW

File: src/hooks/useAuth.tsx [hunk 1]
```diff
@@ -3,34 +3,48 @@ import {
...
+    onAuthStateChanged(auth, (user) => {
+      if (user) {
+        setIsLogged(true);
+        // console.log("uid: " + user.uid);
+      } else {
+        setIsLogged(false);
+        // console.log("user is not logged");
+      }
+    });
+  };
```
Comment (masterHAWK99) [inline src/hooks/useAuth.tsx:25]: It should be declared only once and  listen for changes.

File: src/hooks/useAuth.tsx [hunk 2]
```diff
@@ -3,34 +3,48 @@ import {
...
+      console.log(err);
     }
   };
 
   const login = async () => {
     try {
       await signInWithEmailAndPassword(auth, email, password);
     } catch (err) {
-      alert(err);
+      console.log(err);
```
Comment (masterHAWK99) [inline src/hooks/useAuth.tsx:39]: I tried to use it to login user and there's error because this function gets empty strings.

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

File: src/hooks/useAuth.tsx (hunk 2 above)
Comment (GalacticOctopus) [reply src/hooks/useAuth.tsx:39]: You might've supplied it with empty string, but it definitely works if all is inputted correctly

File: .gitignore [hunk 3]
```diff
@@ -21,7 +21,4 @@ dist-ssr
 *.ntvs*
 *.njsproj
 *.sln
-*.sw?
-
-# Environment
-.env
\ No newline at end of file
+*.sw?
```
Comment (masterHAWK99) [inline .gitignore]: Push .env file without values and add this file to .gitignore again.

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

**Pull request merged**